pkg/stanza/fileconsumer/emit/emit.go

```diff
@@ -7,4 +7,16 @@ import (
 	"context"
 )
 
-type Callback func(ctx context.Context, token []byte, attrs map[string]any) error
+type Callback func(ctx context.Context, token Token) error
```

Comment:
I considered if this should be a `*Token`, but since the Token struct only contains two "reference type" members, a slice and a map, I thought it wouldn't help much to create a pointer to a struct that contains two pointers? :thinking: 

Please let me know if I'm missing anything. My Go language skills are still limited.

Reply:
Seems reasonable to me. This is technically a breaking change, but I think it obscure enough code that we should only note it as a changelog.

Reply:
Right, thanks for pointing this out. I'll out an API changelog entry for this.